pkg/controller/directvolumemigration/rsync.go

```diff
@@ -693,13 +682,14 @@ func (t *Task) createPVProgressCR() error {
 				},
 				Status: migapi.DirectVolumeMigrationProgressStatus{},
 			}
-			err := t.Client.Create(context.TODO(), &dvp)
+			migapi.SetOwnerReference(t.Owner, t.Owner, &dvmp)
+			err := t.Client.Create(context.TODO(), &dvmp)
 			if k8serror.IsAlreadyExists(err) {
-				t.Log.Info("Rsync client progress CR already exists on destination", "namespace", dvp.Namespace, "name", dvp.Name)
+				t.Log.Info("Rsync client progress CR already exists on destination", "namespace", dvmp.Namespace, "name", dvmp.Name)
```

Comment:
@alaypatel07 I noticed that this gets logged repeatedly during the migration. Do we really need to log this? What can the user do with this info?

Reply:
@djwhatle this was just convention copied from directvolumemigration controller, if we need this fixed, can we do that as a follow up with both controller at the same time?

Reply:
That works.